src/native/ceph_s3select/example.js

```diff
@@ -0,0 +1,30 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const nb_native = require('../../util/nb_native');
+const { Transform } = require('readable-stream');
+
+class S3SelectStream extends Transform {
+
+    constructor(opts = {}) {
+        super(opts);
+        var context = {query: process.argv[2], input_format: 'csv', row_delimiter: '\n', column_delimiter: ',', use_header_info: false, quote_fields_always: false};
+        this.CephS3lect = new (nb_native().CephS3lect)(context);
```

Comment:
variables/members/functions should be named in snake-case - meaning `this.ceph_s3_select = ...`.